CHANGELOG.md

```diff
@@ -15,7 +15,7 @@
 
 ##### `Filtering Algorithms`
```

Comment:
Since the interface of class `ParticleFilter` heavily changed, maybe it is better to explicitly indicated that some methods were removed and the constructor has changed. What do you think?

Reply:
Done!